net.x_y.ykhwong.dosbox-daum.yml

```diff
@@ -0,0 +1,85 @@
+  - --share=ipc      # needed for X11
+  - --share=network  # Needed for IPX and serial over TCP/IP
+  - --socket=x11     # default to X11, as Gnome does not provide SSD
+  - --socket=pulseaudio
+  - --filesystem=home
+cleanup:
+  - '/include'
+  - '/lib/pkgconfig'
+  - '/share/aclocal'
+  - '/share/man'
+  - '*.la'
+  - '*.a'
+  - '/share/doc'
+
+modules:
+  - "shared-modules/glu/glu-9.json"
+  - "shared-modules/glew/glew.json"  
+  - name: SDL-1.2.15-hq
+    buildsystem: autotools
+    sources:
+      - type: git
+        url: https://github.com/ebenbruyns/sdl1.2.15-hq.git
+        tag: release-1.2.15-hq
+  - "shared-modules/SDL/SDL_net-1.2.8.json"
+  - name: libtbb
+    buildsystem: simple
+    build-commands:
+      - make -e CXXFLAGS="-std=c++14" # -stdlib=libstdc++" 
+      - install -m755 -t /app/lib/ build/linux_*_release/*
+      - install -m755 -t /app/lib/ build/linux_*_debug/concurrent_hash_map.o # the release build of this file crashes on stripping - seems to work
+      - install -D -t /app/include/tbb/ include/tbb/*.h
+      - install -D -t /app/include/serial/tbb include/serial/tbb/*
+      - install -D -t /app/include/tbb/compat include/tbb//compat/*
+      - install -D -t /app/include/tbb/machine include/tbb/machine/*
+      - install -D -t /app/include/tbb/internal include/tbb/internal/*
+    sources:
+      - type: git
+        url: https://github.com/oneapi-src/oneTBB.git
+        tag: 2017_U8
+  - name: open-glide
+    buildsystem: autotools
+    sources:
+      - type: git
+        url: https://github.com/voyageur/openglide.git
+        commit: 1ecc60a83d2255bb9ce8a3e53a5c6bcc7dce44b1   
+  - name: dosbox-daum
+    buildsystem: simple
+    build-commands:
+      - sh autogen.sh
+      - cp gl.h /app/include/GL/  
+      - ./configure --enable-core-inline --prefix=/app
+      - make -e CXXFLAGS="-std=c++11 -fpermissive -I/app/include/SDL -ltbb"
+      - make install
+    sources:
+      - type: git
+        url: https://github.com/ykhwong/dosbox-svn-daum.git
+        commit: 690080fd63ec3877d110240ccc7f0890e272c984
+          #branch: master
+      - type: file
+        path: ./gl.h
+      - type: patch
+        path: flatpak.patch
+      - type: file
+        path: dosboxsvnda.png
+      - type: file
+        path: net.x_y.ykhwong.dosbox-daum.desktop
+      - type: file
+        path: 32x32.png    
+      - type: file
+        path: 48x48.png    
+      - type: file
+        path: 128x128.png    
+    post-install:
+      - install -Dm644 32x32.png /app/share/icons/hicolor/32x32/apps/${FLATPAK_ID}.png
+      - install -Dm644 48x48.png /app/share/icons/hicolor/48x48/apps/${FLATPAK_ID}.png
+      - install -Dm644 128x128.png /app/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png
+      - install -Dm644 ${FLATPAK_ID}.desktop -t /app/share/applications/
+      - desktop-file-edit --set-key=Icon --set-value=${FLATPAK_ID} /app/share/applications/${FLATPAK_ID}.desktop
```

Comment:
I followed the dosbox x manifest as an example... thought it was acceptable, I'll change that...

Reply:
no you didn't. because dosbox doesn't have a .desktop that in the flathub repo or generate by some script.

it does it because it has to modify what's coming upstream, in which case it's the way to go.

you just take things out of context.

Reply:
I was just explaining what I saw, so it turns out I was wrong, no big deal, like I said I'll change that.